src/panel/service/ServicePanelDesktop.jsx

```diff
@@ -1,21 +1,40 @@
 import React, { Fragment, useState } from 'react';
 import { Panel, Chevron } from 'src/components/ui';
 import CategoryList from 'src/components/CategoryList';
-import { useI18n } from 'src/hooks';
+import { useConfig, useI18n } from 'src/hooks';
 import { Flex, Button } from '@qwant/qwant-ponents';
 
 const ServicePanelDesktop = () => {
   const [collapsed, setCollapsed] = useState(true);
   const { _ } = useI18n();
+  const ecoResponsible = useConfig('features').ecoResponsible;
 
   return (
     <Fragment>
       <Panel className="service_panel u-mb-m">
         <h3 className="u-text--smallTitle u-mb-s">
           {_('Search around this place', 'service panel')}
         </h3>
-        <CategoryList className="service_panel__categories" limit={collapsed ? 4 : undefined} />
+        <CategoryList
+          className="service_panel__categories"
+          ecoResponsible={false}
+          limit={collapsed ? 4 : undefined}
+        />
       </Panel>
+      {ecoResponsible && (
+        <>
```

Comment:
is it on purpose ?

Reply:
Oh you're right, I'm not sure how it slipped here as there is only one child component, *Panel*, it may just be due to an older attempt at implementing this :shrug: